package.json

```diff
@@ -5,17 +5,22 @@
   "bin": {
     "contentful-export": "./bin/contentful-export"
   },
-  "main": "index.js",
```

Comment:
I would keep `index.js` as a main entry point so that way we have more control over what to expose and what not